django/core/paginator.py

```diff
@@ -96,19 +95,35 @@ def _check_object_list_is_ordered(self):
         """
         Warn if self.object_list is unordered (typically a QuerySet).
         """
-        if hasattr(self.object_list, 'ordered') and not self.object_list.ordered:
+        if hasattr(self.object_list,
+                   'ordered') and not self.object_list.ordered:
```

Comment:
I don't think we want to shorten this line. Django allows [up to 119 characters](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/).